From 6aa217916888ee02aae1292e7eb8cb98f5f86357 Mon Sep 17 00:00:00 2001
From: Gustavo Duarte <gduarte@redhat.com>
Date: Mon, 1 Sep 2014 20:15:31 -0400
Subject: [powerpc] perf/hv-gpci: Make device attr static

Message-id: <1409602534-23951-13-git-send-email-gduarte@redhat.com>
Patchwork-id: 90125
O-Subject: [RHEL7.1 PATCH BZ 947159 12/15] powerpc/perf/hv-gpci: Make device attr static
Bugzilla: 947159
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=947159
Upstream Status: 58a685c2d8f0616b5c6cfa0f7a9f2d081af3c6fd

commit 58a685c2d8f0616b5c6cfa0f7a9f2d081af3c6fd
Author: Cody P Schafer <cody@linux.vnet.ibm.com>
Date:   Tue Apr 15 10:10:52 2014 -0700

    powerpc/perf/hv-gpci: Make device attr static

    Signed-off-by: Cody P Schafer <cody@linux.vnet.ibm.com>
    Signed-off-by: Benjamin Herrenschmidt <benh@kernel.crashing.org>

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/arch/powerpc/perf/hv-gpci.c b/arch/powerpc/perf/hv-gpci.c
index 8fee1dc..c9d399a 100644
--- a/arch/powerpc/perf/hv-gpci.c
+++ b/arch/powerpc/perf/hv-gpci.c
@@ -78,7 +78,7 @@ static ssize_t kernel_version_show(struct device *dev,
  return sprintf(page, "0x%x\n", COUNTER_INFO_VERSION_CURRENT);
 }
 
-DEVICE_ATTR_RO(kernel_version);
+static DEVICE_ATTR_RO(kernel_version);
 HV_CAPS_ATTR(version, "0x%x\n");
 HV_CAPS_ATTR(ga, "%d\n");
 HV_CAPS_ATTR(expanded, "%d\n");
-- 
1.7.1